src/reducers/rootReducer.js

```diff
@@ -1,5 +1,8 @@
 import { combineReducers } from "redux";
 import simpleReducer from "./simpleReducer";
+import userReducer from "../reducers/authReducers/socialAuthReducer";
 export default combineReducers({
-  simpleReducer
+  simpleReducer,
+  userReducer
+
```

Comment:
Remove this extra space. Try to proof read and clean up your code in all other files

Reply:
@LehruAngela  I have made this change